ethminer/MinerAux.h

```diff
@@ -627,9 +633,11 @@ class MinerCLI
 			<< "        1: eth-proxy compatible: dwarfpool, f2pool, nanopool (required for hashrate reporting to work with nanopool)" << endl
 			<< "        2: EthereumStratum/1.0.0: nicehash" << endl
 			<< "    -RH, --report-hashrate Report current hashrate to pool (please only enable on pools supporting this)" << endl
+			<< "    -HWMON Displays gpu temp and fan percent." << endl
```

Comment:
@MariusVanDerWijden: I think this is wrong here?

Reply:
This was the old version, I didn't change it... It's a merge error. I'll resolve it